docker-base-runtime/run-and-cleanup

```diff
@@ -20,13 +21,25 @@ def main():
         description='Run a command and delete a path when it terminates. '
                     'Signals are forwarded to the executed process.',
         usage='%(prog)s path -- command [args...]')
+    parser.add_argument('-c', '--create', action='store_true', help='Create the path')
+    parser.add_argument('-t', '--tmp', action='store_true', help='Set as TMPDIR for child')
     parser.add_argument('path', help='Path to clean up')
     parser.add_argument('command', nargs='+', help='Command to run')
     args = parser.parse_args()
+
+    if args.create:
+        try:
+            os.mkdir(args.path)
```

Comment:
In the spirit of Python3, and allowing recursive dirs, how about pathlib.Path(args.path).mkdir(parents=True, exist_ok=True) ?

Reply:
I was undecided about creating parents. Given the use cases I would be surprised if the parent doesn't exist, and bumbling on might be hiding deeper errors. But on the other hand if it doesn't exist then it's pain to get it created, so I'll add support.

Reply:
Done.